bitnami/sonarqube/templates/install_plugins.yaml

```diff
@@ -7,11 +7,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: {{ printf "%s-install-plugins-configmap" (include "common.names.fullname" .) }}
-  namespace: {{ .Release.Namespace | quote }}
```

Comment:
I think you removed this by mistake.

Reply:
fixed at https://github.com/bitnami/charts/pull/18748/commits/c0fd1c53659ea4bf6c1a4eddbb46fda8406aaf5e